frigate/detectors/detector_config.py

```diff
@@ -39,6 +39,8 @@ class ModelTypeEnum(str, Enum):
     rfdetr = "rfdetr"
     ssd = "ssd"
     yolox = "yolox"
+    yolov9 = "yolov9"
```

Comment:
we should use the yolo-generic here. Yolov9 was removed in a recent PR so that all yolo variants can be handled under one model type. Also see https://github.com/blakeblackshear/frigate/pull/10717 related to yolov8, I mentioned this above as well.